conda/core/portability.py

```diff
@@ -194,7 +194,7 @@ def replace(match):
         return match.group().replace(search, replacement) + b"\0" * padding
 
     original_data_len = len(data)
-    pat = re.compile(re.escape(search) + b"(?:(?!(?:" + zeros + b")).)*" + zeros)
+    pat = re.compile(re.escape(search) + b"(?:(?!(?:" + zeros + b"))[\\s\\S])*" + zeros)
```

Comment:
```suggestion
    pat = re.compile(re.escape(search) + b"(?:(?!(?:" + zeros + b")).)*" + zeros, flags=re.DOTALL)
```

Would this read clearer? i.e. being explicit about "I want `.` to match everything including newlines"?

Reply:
Yes, I think so! Added it.